src/test/pmem2_map_prot/pmem2_map_prot.c

```diff
@@ -273,15 +318,292 @@ test_rw_mode_none_prot(const struct test_case *tc,
 	return 1;
 }
 
+/*
+ * sum_asm[] --> simple program in assembly which calculates '2 + 2' and
+ * returns the result
+ */
+unsigned char sum_asm[] = {
```

Comment:
All tests that use this must `@t.require_architectures('x86_64')`.

Are you sure the behavior of this thing is defined?